src/components/rockets/RocketsPage.js

```diff
@@ -0,0 +1,7 @@
+// // import { useEffect } from 'react';
+// import RocketsFalcon from './RocketsFalcon';
+// // import { fetchRockets } from '../../redux/rockets/rocketsSlice';
+
+// const RocketsPage = () => (<RocketsFalcon />);
+
+// export default RocketsPage;
```

Comment:
Hi Marko, please remove this file as its entire content is commented out. Please check if the app functionality is not affected after you remove this file